WordPress/Classes/Services/PostService+Revisions.swift

```diff
@@ -27,9 +27,7 @@ extension PostService {
                                         let revisions = self.syncPostRevisions(from: remoteRevisions ?? [],
```

Comment:
I haven't loaded the code locally, but would this result in an unused assignment warning now?

Reply:
Good catch. There is indeed a warning. Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20267/commits/b71255d9b7396e769298b3a9226357c1f1695fe5